src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -137,6 +137,6 @@ file that was distributed with this source code.
     {% if sonata_admin.field_description.mappingtype == 'many' %}
         {{ block('sonata_admin_mongo_collection') }}
     {% else %}
-        INVALID MODE : {{ id }} - type : sonata_type_collection - mapping : {{ sonata_admin.field_description.mappingtype }}
+        INVALID MODE : {{ id }} - type : CollectionType - mapping : {{ sonata_admin.field_description.mappingtype }}
```

Comment:
We should use `constant()` in order to show the real name.

Reply:
Thanks @phansys for the review! in this case, I'm not sure if `constant()` with `CollectionType::class` would work since it's not a constant. To be honest, I'm not even sure if it's usefull that else or we can leave it just like above:

`INVALID MODE : {{ id }}`

Reply:
It's fine for me. I can even confirm the pseudo constant `::class` is not working with `constant()`:
```twig
{{ constant('\\stdClass::class') }}
{# "Warning: constant(): Couldn't find constant \stdClass::class" #}
```